packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -65,6 +66,43 @@ def vip_app(container: marathon.Container, network: marathon.Network, host: str,
         raise AssertionError('Unexpected network: {}'.format(network.value))
 
 
+def pod_vip_app(network: marathon.Network, host: str, vip: str):
+    ''' Creates a single app in docker container in pod
+    '''
+    container_port = 0
+    if network == marathon.Network.USER:
+        container_port = unused_port(marathon.Network.USER)
+    test_uuid = uuid.uuid4().hex
+    app = {
+        'id': '/integration-test-{}'.format(test_uuid),
+        'placement': {'acceptedResourceRoles': ['*', 'slave_public']},
+        'containers': [{
+            'name': 'app-{}'.format(test_uuid),
+            'resources': {'cpus': 0.01, 'mem': 8},
```

Comment:
we should probably set these resources a bit higher, else the apps itself might fail due to OOM. CPU: 0.1, Mem: 128